QUEST_LV_0100.tsv

```diff
@@ -3839,10 +3839,10 @@ QUEST_LV_0100_20150317_003838	Check status of the Scout
 QUEST_LV_0100_20150317_003839	Please check the status of the attack Scout to Kepa.
 QUEST_LV_0100_20150317_003840	Remind the Scout that there is an order to assemble to Klaipeda
 QUEST_LV_0100_20150317_003841	Remind the Scout again that there is an order to assemble to Klaipeda.
-QUEST_LV_0100_20150317_003842	Defeat the Leaf Bugs and collect the luggages.
-QUEST_LV_0100_20150317_003843	The Scout says he can't return until he finds his luggages. Find the Scout's stolen luggages from the Leaf Bugs.
-QUEST_LV_0100_20150317_003844	Give the luggages to the Scout
-QUEST_LV_0100_20150317_003845	Return the luggages to the Scout and tell him to return to Klaipeda. 
+QUEST_LV_0100_20150317_003842	Defeat the Leaf Bugs at Uoros farm and collect the belongings.
```

Comment:
I just started this quest again, and it will tell you to press 'M' to open the map to check the location. So, adding 'at Uoros Farm' isn't really necessary, is it?